src/e2e/pages/dashBoardPage.ts

```diff
@@ -0,0 +1,23 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Locator, Page } from "@playwright/test";
+
+export class DashboardPage {
+  readonly page: Page;
+  readonly dataBreachEmailDropdown: Locator;
+  readonly siteFoundImage: Locator;
+  readonly breachStats: Locator;
+
+  constructor(page: Page) {
+    this.page = page;
+    this.dataBreachEmailDropdown = page.locator("custom-select");
```

Comment:
Thinking out loud here.. as a part of our smoke tests, we'd want to cover all locators so we can detect any that's returning null?

Reply:
I wonder if there's a way to check for coverage